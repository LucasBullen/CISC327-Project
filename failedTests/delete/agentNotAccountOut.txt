2c2
< Logged into agent mode.
---
> Logged into atm mode.
4c4
< Account does not exists. Delete cancelled.
---
> Account does not exist. Delete cancelled.
