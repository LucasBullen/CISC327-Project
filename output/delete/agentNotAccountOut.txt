Select mode: atm or agent.
Logged into agent mode.
Enter account number to delete.
Account does not exists. Delete cancelled.
Logged out.
