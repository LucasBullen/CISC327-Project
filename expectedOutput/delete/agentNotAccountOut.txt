Select mode: atm or agent.
Logged into atm mode.
Enter account number to delete.
Account does not exist. Delete cancelled.
Logged out.
